rust/gitxetcore/Cargo.toml

```diff
@@ -79,7 +79,7 @@ nfsserve = "0.10"
 atty = "0.2"
 libc = "0.2"
 itertools = "0.10.5"
-shellish_parse = "2.2"
+snailquote = "0.3.1"
```

Comment:
Likely an incorrect merge happened, this reverted https://github.com/xetdata/xet-core/pull/416